### PR TITLE
dash/a11y-minimal-demo

### PR DESCRIPTION
**Pull request by sebastianbochan**
Added a11y descriptions to parts of the minimal demo.

### CODE REVIEW

File: samples/dashboards/demo/minimal/demo.js [hunk 1]
```diff
@@ -162,6 +187,18 @@ Dashboards.board('container', {
             },
             title: {
                 text: ''
+            },
+            tooltip: {
+                valueSuffix: ' mcg',
+                stickOnContact: true
+            },
+            accessibility: {
+                chartContainerLabel: 'Iron',
```
Comment (marvin19) [inline samples/dashboards/demo/minimal/demo.js:196]: - It looks like the Vitamin A and Iron is switched for the demos,
- chartContainerLabel belongs in `lang`

File: samples/dashboards/demo/minimal/demo.js [hunk 2]
```diff
@@ -162,6 +187,18 @@ Dashboards.board('container', {
...
+                typeDescription: 'Column chart',
+                description: 'The iron amount in Foods',
```
Comment (marvin19) [inline samples/dashboards/demo/minimal/demo.js:198]: It looks like the Vitamin A and Iron is switched for the demos

File: samples/dashboards/demo/minimal/demo.js [hunk 3]
```diff
@@ -222,6 +262,18 @@ Dashboards.board('container', {
                 animation: false,
                 type: 'column',
                 spacing: [30, 30, 30, 20]
+            },
+            tooltip: {
+                valueSuffix: ' mcg',
+                stickOnContact: true
+            },
+            accessibility: {
+                chartContainerLabel: 'Vitamin A',
```
Comment (marvin19) [inline samples/dashboards/demo/minimal/demo.js:271]: - It looks like the Vitamin A and Iron is switched for the demos
- chartContainerLabel belongs in `lang`

File: samples/dashboards/demo/minimal/demo.js [hunk 4]
```diff
@@ -222,6 +262,18 @@ Dashboards.board('container', {
...
+                typeDescription: 'Column chart',
+                description: 'The vitamin A amount in Foods',
```
Comment (marvin19) [inline samples/dashboards/demo/minimal/demo.js:273]: It looks like the Vitamin A and Iron is switched for the demos

File: samples/dashboards/demo/minimal/demo.js [hunk 5]
```diff
@@ -122,7 +144,10 @@ Dashboards.board('container', {
         },
         chartOptions: {
             xAxis: {
-                type: 'category'
+                type: 'category',
+                accessibility: {
+                    description: 'Grocieries'
```
Comment (marvin19) [inline samples/dashboards/demo/minimal/demo.js:149]: Typo --> Groceries

**Review by marvin19 — CHANGES_REQUESTED** [review]

File: samples/dashboards/demo/minimal/demo.js [hunk 6]
```diff
@@ -192,10 +192,15 @@ Dashboards.board('container', {
...
+                    chartContainerLabel: 'Vitamin A in food. Highcharts Interactive Chart.'
+                }
+            },
             accessibility: {
-                chartContainerLabel: 'Iron',
-                typeDescription: 'Column chart',
-                description: 'The iron amount in Foods',
+                description: `The chart is displaying the vitamin A amount in
+                micrograms for some groceries. There is a plotLine demonstrating
+                the daily Recommended Dietary Allowance (RDA)`,
```
Comment (marvin19) [inline samples/dashboards/demo/minimal/demo.js:203]: Can you also put in the value of the RDA?

File: samples/dashboards/demo/minimal/demo.js [hunk 7]
```diff
@@ -267,10 +272,15 @@ Dashboards.board('container', {
...
+                    chartContainerLabel: 'Iron in food. Highcharts Interactive Chart.'
+                }
+            },
             accessibility: {
-                chartContainerLabel: 'Vitamin A',
-                typeDescription: 'Column chart',
-                description: 'The vitamin A amount in Foods',
+                description: `The chart is displaying the Iron amount in
+                micrograms for some groceries. There is a plotLine demonstrating
+                the daily Recommended Dietary Allowance (RDA)`,
```
Comment (marvin19) [inline samples/dashboards/demo/minimal/demo.js:283]: Can you also put in the value of the RDA?

**Review by marvin19 — CHANGES_REQUESTED** [review]
Looks good!

File: samples/dashboards/demo/minimal/demo.js [hunk 8]
```diff
@@ -72,9 +72,31 @@ Dashboards.board('container', {
                         }]
                     }
                 }, {
-                    id: 'dashboard-col-0'
+                    id: 'dashboard-col-0',
+                    responsive: {
+                        small: {
+                            width: '100%'
+                        },
+                        medium: {
```
Comment (karolkolodziej) [inline samples/dashboards/demo/minimal/demo.js:80]: IMO it looks too small in medium.

File: ts/Dashboards/cypress/support/commands.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (karolkolodziej) [inline ts/Dashboards/cypress/support/commands.js:1]: Are these files related to the fix?

**Review by karolkolodziej — COMMENTED** [review]
Can you please investigate if we can do sth about the tooltips:
image
It is visible when you load the page and start to hover over one of the charts.

**Pull request closed without merging**